programs/stake/src/stake_state.rs

```diff
@@ -2386,7 +2423,8 @@ mod tests {
                     points: 1
                 },
                 &vote_state,
-                None
+                None,
+                &mut None::<fn(&_)>,
```

Comment:
could we have a default "null" tracer to avoid the Option.  Just thinking that a `&mut null_tracer` construct would look nicer than the borderline unreadable (a non Rust expert) `&mut None::<fn(&_)>` construct

Reply:
ref: 707c276699 9712389566